server/portal/apps/projects/management/commands/migrate-projects.py

```diff
@@ -0,0 +1,51 @@
+"""Management command."""
+
+from django.core.management.base import BaseCommand
+from portal.libs.agave.utils import service_account
+from portal.apps.projects.models.base import Project
+from portal.apps.projects.models.metadata import AbstractProjectMetadata
+from django.contrib.auth import get_user_model
+import logging
+
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """Command class."""
+
+    help = (
+        'Reconcile projects from CEPv1 migration that do not have '
+        'a PI assigned. This will attempt to assign the creator of '
+        'these projects as PI.'
+    )
+
+    def handle(self, *args, **options):
+        """Handle command."""
+        agc = service_account()
+        metadatas = AbstractProjectMetadata.objects.all().filter(pi=None)
+        for meta in metadatas:
+            try:
+
+                project = Project(agc, meta.project_id)
+                storage = project.storage
+                roles = storage.roles.to_dict().items()
+                admins = list(
+                    filter(
+                        lambda role_tuple: role_tuple[0] != 'wma_prtl' and (role_tuple[1] == 'ADMIN' or role_tuple[1] == 'OWNER'),
+                        roles
+                    )
+                )
+                if len(admins) != 1:
```

Comment:
How many existing projects do you predict will have multiple admins? Wondering if this is common enough that we should have a backup plan for them.

Reply:
> How many existing projects do you predict will have multiple admins? Wondering if this is common enough that we should have a backup plan for them.

That's difficult to say without pulling all the projects in agave and inspecting all the roles, but the code shouldn't allow it. The only case in which that would happen is if someone created an ownerless project in the portal and then went back to the tapis cli and manually assigned themselves an admin privilege.